WalletWasabi.Tests/UnitTests/AllFeeEstimateTests.cs

```diff
@@ -218,18 +268,37 @@ private static MockRpcClient CreateAndConfigureRpcClient(Func<int, FeeRate> esti
 					}),
 				OnGetPeersInfoAsync = async () =>
 					await Task.FromResult(hasPeersInfo ? new[] { new PeerInfo() } : Array.Empty<PeerInfo>()),
-
-				OnGetMempoolInfoAsync = async () =>
-					await Task.FromResult(new MemPoolInfo
-					{
-						MemPoolMinFee = memPoolMinFee // 1 s/b (default value)
-					}),
+				OnGetMempoolInfoAsync = async () => await Task.FromResult(ParseMemPoolInfo(memPoolMinFee)),
 				OnEstimateSmartFeeAsync = async (target, _) =>
 					await Task.FromResult(new EstimateSmartFeeResponse
 					{
 						Blocks = target,
 						FeeRate = estimator(target)
 					})
 			};
-	}
+
+		private static MemPoolInfo ParseMemPoolInfo(double memPoolMinFee)
```

Comment:
```suggestion
		private static MemPoolInfo ParseMempoolInfo(double memPoolMinFee)
```

Typically, it is "mempool", I believe.

Reply:
Right.

Reply:
Done.